drivers/adcxx1c/adcxx1c_saul.c

```diff
@@ -26,7 +26,9 @@
 
 static int read_adc(const void *dev, phydat_t *res)
 {
-    adcxx1c_read_raw((const adcxx1c_t *)dev, res->val);
+    if (adcxx1c_read_raw((const adcxx1c_t *)dev, res->val)) {
```

Comment:
Some people tend to think `&res->val[0]` is more explicit.

Reply:
will fix

Reply:
Done